### PR TITLE
Zeppelin-472 ] Shortcuts execution bugs.

### PR DESCRIPTION
**Pull request by cloverhearts**
https://issues.apache.org/jira/browse/ZEPPELIN-472

If you use the Shift + Enter keys to execute the Paragraph,
If you continue to run while Paragraph enter the Shift + Enter key,
Paragraph enter that this is done constantly being requested RUNNING state.

Case)

Pragraph details

Scala) Thread.sleep (1000);
Key press running SHIFT + Enter * 10

### CODE REVIEW

File: zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js [hunk 1]
```diff
@@ -587,7 +587,10 @@ angular.module('zeppelinWebApp')
         exec: function(editor) {
           var editorValue = editor.getValue();
           if (editorValue) {
-            $scope.runParagraph(editorValue);
+            if (!($scope.paragraph.status === 'RUNNING' || $scope.paragraph.status === 'PENDING'))
+            {
```
Comment (Leemoonsoo) [inline zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js]: This is minor. but we maybe want to keep the same code style. ie.

```
if ( ... ) {
  ...
}
```

**Comment by Leemoonsoo** [comment]
Looks good to me.

File: zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js (hunk 1 above)
Comment (cloverhearts) [inline zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js]: Okay.

Thank you, feedback.

**Comment by Leemoonsoo** [comment]
Thanks for the updating code style. 
Merge if there're no more discussions.

**Comment by jongyoul** [comment]
LGTM

**Comment by cloverhearts** [comment]
Thank you for all your comments.

**Pull request closed without merging**